lib/workers/repository/extract/index.ts

```diff
@@ -47,6 +47,16 @@ export async function extractAllDependencies(
   const extractResults = await Promise.all(
     extractList.map(async (managerConfig) => {
       const packageFiles = await getManagerPackageFiles(managerConfig);
+      for (const p of packageFiles ?? []) {
+        if (p.deps) {
+          for (const dep of p.deps) {
```

Comment:
```suggestion
          for (const dep of p.deps ?? []) {
```
and remove the if, so even less nesting.

@rarkins should we make `deps` required? i think almost all managers are setting that anyways and package file would be null instead. 🤔

Reply:
I think it would be OK to make it required, but possible to be an empty array. There's some cases where it's useful to note that we detected a definite package file even if it has no deps (rather than confuse people as to whether it was detected).